src/inputs/datePickerInput/datePickerInput.jsx

```diff
@@ -361,7 +363,7 @@ class DatePickerInput extends React.PureComponent {
 
         if (isNotDisabled) {
             // eslint-disable-next-line no-underscore-dangle, react/no-find-dom-node
-            ReactDOM.findDOMNode(this._datePickerInput._input).focus();
+            this.datePickerInput.current.inputElement.focus();
```

Comment:
looks like the eslint directive above is no longer necessary

Reply:
Fixed.